disconnected/mirroring/installing-mirroring-disconnected.adoc

```diff
@@ -10,6 +10,11 @@ Running your cluster in a restricted network without direct internet connectivit
 
 You can use the oc-mirror OpenShift CLI (`oc`) plugin to mirror images to a mirror registry in your fully or partially disconnected environments. You must run oc-mirror from a system with internet connectivity in order to download the required images from the official Red Hat registries.
 
+[IMPORTANT]
+====
+The `oc-mirror` plugin v1, which runs when the `--v2` flag is not set, is now deprecated. It is recommended to transition to xref:../../disconnected/mirroring/about-installing-oc-mirror-v2.adoc#installation-oc-mirror-v2-about_about-installing-oc-mirror-v2[oc-mirror plugin v2] for continued support and improvements.
```

Comment:
```suggestion
The oc-mirror plugin v1 is now deprecated. Transition to the xref:../../disconnected/mirroring/about-installing-oc-mirror-v2.adoc#installation-oc-mirror-v2-about_about-installing-oc-mirror-v2[oc-mirror plugin v2] for continued support and improvements.
```


- As per the jira description, this part is not needed hence suggest to remove that part. _"which runs when the `--v2` flag is not set"._
- We don't use back ticks for "oc-mirror  plugin".
- Avoid using "recommend". See https://redhat-documentation.github.io/supplementary-style-guide/#recommend